### ISSUE TITLE
[Bug] Move file command does not work when destination folder contains file with same name

### ISSUE THREAD

**Issue by isdmg** [tadashi-aikawa/obsidian-another-quick-switcher#260]
### Check for Commonly Missed Points

- [X] Files not shown in the suggestions are not an active file. (An active file is not shown in the suggestions.)

### Summary

The `Move file to another folder` command does not work when the destination folder contains a file with the same name as the current file.

### Steps to Reproduce in the Sandbox Vault

Assume that we have the following file structure:

folder1/
└── same-name.md
folder2/
└── same-name.md

If we run the `Move file to another folder` command on `same-name.md` in `folder1` and choose the destination folder to be `folder2`, we get the following error in the developer console:

```js
Uncaught (in promise) Error: Destination file already exists!
    at e.<anonymous> (app.js:1:536060)
    at app.js:1:237228
    at Object.next (app.js:1:237333)
    at a (app.js:1:236051)
```

### Expected Behavior

The native Obsidian command `Move current file to another folder` moves the current file by appending a numerical value on the file name.

I expect the `Move file to another folder` command to copy the behavior of `Move current file to another folder` of Obsidian:

**Before the command**
folder1/
└── same-name.md
folder2/
└── same-name.md

**After the command**
folder1/
└── (EMPTY)
folder2/
├── same-name.md
└── ***same-name 1.md***

### Actual Behavior

Currently, the plugin does not work as expected. If we run the `Move file to another folder` command on `same-name.md` in `folder1` and choose the destination folder to be `folder2`, we get this behavior:

**Before the command**
folder1/
└── same-name.md
folder2/
└── same-name.md

**After the command**
folder1/
└── same-name.md
folder2/
└── same-name.md

### Obsidian Version

1.7.4

### Another Quick Switcher Version

12.0.1

### OS

Windows

**Comment by tadashi-aikawa** [comment]
Thank you for the bug report, @isdmg :)

**Comment by tadashi-aikawa** [comment]
@isdmg 
I attempted to fix this issue, but I just received an interesting post.

https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/discussions/261

I hadn’t been aware of it, but it seems that Obsidian now has a similar command as a core feature. Since you're using a similar command in Another Quick Switcher, I'd like to hear your thoughts on this: Do you think the Obsidian core feature can fully replace it?

**Comment by isdmg** [comment]
@tadashi-aikawa 
Yes. The core feature fully replaces the command in Another Quick Switcher. However, I like using the Another Quick Switcher command because you have the option to not display the full path of the directory, which is helpful for deeply nested folders.

I also believe it's safe to delete this command. And the issue that I opened isn't really that big of deal for me. You can do what you think is best for the plugin. I'll still use it even without the move file command feature :)

**Comment by tadashi-aikawa** [comment]
@isdmg 
Thank you for the helpful response! As you pointed out, there are indeed display differences. I believe which feature to use is up to the user, so I’ll keep this command as it is. 👍

**Comment by tadashi-aikawa** [comment]
When paths conflict, I've modified it to append a timestamp-based suffix instead of a sequential number. Additionally, a dot is used instead of whitespace.

![image](https://github.com/user-attachments/assets/e84236a2-c168-4600-b955-03cbdf48a702)

**Comment by tadashi-aikawa** [comment]
@isdmg 
Released in [v12.1.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/12.1.0) 🚀
